docs/content/en/docs/reference/templates/tracing_policy.tmpl

```diff
@@ -0,0 +1,127 @@
+---
+title: "Tracing Policy"
+description: "This reference documentation is generated from the Tracing Policy CRD specification, detailing its fields and usage."
+weight: 1
```

Comment:
maybe put 5 for now and we can always reorganize later if needed but that's not super important
```suggestion
weight: 5
```

Reply:
Fixed